### PR TITLE
openvpn-client-export package - code cleanup and fixes

### PR DESCRIPTION
**Pull request by doktornotor**
- add standard XML and copyright headers
- remove bunch of 077 chmods
- remove unused tags

### CODE REVIEW

File: config/openvpn-client-export/openvpnexport.inc [hunk 1]
```diff
@@ -1,4 +1,31 @@
 <?php
+/*
+	openvpnexport.inc
+	part of pfSense (http://www.pfSense.com)
+	Copyright (C) 2011 Marcello Coutinho
```
Comment (rbgarga) [inline config/openvpn-client-export/openvpnexport.inc]: Was Marcelo the only one that touch this file?

File: config/openvpn-client-export/openvpn-client-export.xml [hunk 2]
```diff
@@ -1,12 +1,52 @@
...
+	<![CDATA[
+/* $Id$ */
+/* ====================================================================================== */
+/*
+	openvpn-client-export.xml
+	part of pfSense (http://www.pfSense.com)
+	Copyright (C) 2011-2015 Jim Pingle
+	Copyright (C) 2009-2010 Ermal Luçi
+	Copyright (C) 2009 Scott Ullrich
+	Copyright (C) 2008 Bill Marquette
```
Comment (rbgarga) [inline config/openvpn-client-export/openvpn-client-export.xml]: Where did you get this information?

Comment (doktornotor) [inline config/openvpn-client-export/openvpn-client-export.xml]: Well, that's from commits history. Have better idea? :)

Comment (rbgarga) [inline config/openvpn-client-export/openvpn-client-export.xml]: Nope, that is fine, I was just confirming the method :) Thanks!

File: config/openvpn-client-export/openvpnexport.inc (hunk 1 above)
Comment (doktornotor) [inline config/openvpn-client-export/openvpnexport.inc]: Not really, wrong paste, also wrong website link. Will fix now. ;)

**Pull request merged**